epochStart/metachain/economics.go

```diff
@@ -440,12 +492,11 @@ func (e *economics) adjustRewardsPerBlockWithLeaderPercentage(
 
 // compute inflation rate from genesisTotalSupply and economics settings for that year
 func (e *economics) computeInflationBeforeSupernova(currentRound uint64, epoch uint32) float64 {
-	roundsPerDay := common.ComputeRoundsPerDay(e.roundTime.TimeDuration(), e.enableEpochsHandler, epoch)
-
+	roundsPerDay := numberOfSecondsInDay / uint64(e.roundTime.TimeDuration().Seconds())
```

Comment:
will this be ok for sub-second round? 

e.g. would e.roundTime.TimeDuration().Seconds() return 0 if round is 0.6s ?